docs/docs/running-hello-world.md

```diff
@@ -225,70 +225,19 @@ To generate `host-release.jar`:
     On a Linux machine that meets the [system requirements](enclave-modes.md#system-requirements), you can run your
     app in all the modes the same way as in mock mode.
 
-=== "macOS"
+=== "macOS/Windows"
 
-    You need a Linux environment to run the host in simulation mode. This section describes how to create such an
-    environment using Conclave's Docker integration.
+    Conclave needs a Linux environment to run the host in simulation mode. But you don't need to do anything as long as
+    Docker is installed on your system. You can run the host and client as we did for mock mode. The only differences are that the host will be
+    run from inside the container, and the host JAR will be named `host-simulation.jar` rather than `host-mock.jar`.
```

Comment:
`will be run from inside`. Why not `will run inside`?

Reply:
Please rewrite as: "The difference is that you must run the host from inside the container, and the host JAR is named `host-simulation.jar` rather than `host-mock.jar`.